src/test/java/ArrayStackTest2.java

```diff
@@ -0,0 +1,243 @@
+import org.junit.jupiter.api.*;
+
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class ArrayStackTest2 {
+
+    private Stack<Integer> classUnderTest;
+    private Stack<Integer> preState;
+
+//    @Test
```

Comment:
Just remove imo. Think it's confusing to leave it commented out. We have tests in the assignments repo, so we're confident enough the implementation is correct.